packages/blockchain-wallet-v4/src/types/HDWallet.js

```diff
@@ -66,7 +66,7 @@ export const deriveAccountNodeAtIndex = (seedHex, index, network) => {
 
 export const generateAccount = curry((index, label, seedHex) => {
   let node = deriveAccountNodeAtIndex(seedHex, index, Bitcoin.networks.bitcoin)
-  return HDAccount.fromJS(HDAccount.js(`${label} ${index + 1}`, node))
+  return HDAccount.fromJS(HDAccount.js(label, node))
```

Comment:
What happens on wallet recovery? Is there a label defined?

Reply:
On wallet recovery the labels are defined in the js method of types/HDWallet.js with ```HDAccount.js(`${label} ${i + 1}`, node(i), xpub)``` not in generateAccount